src/pages/fr/core-concepts/astro-pages.md

```diff
@@ -67,13 +68,13 @@ Ceci est ma page, écrite en **Markdown.**
 
 ## Pages non-HTML
 
-Des pages qui ne sont pas du HTML, comme des `.json` ou des `.xml`, ou même des fichiers non-textuels comme des images peuvent être générées à partir de **Routes de Fichiers**.
+Des pages qui ne sont pas du HTML, comme des `.json` ou des `.xml`, ou même des fichiers, tel que des images, peuvent être générées à partir de chemins API ou appellés couramment "**Routes de Fichiers**".
 
-Les **Routes de Fichiers** doivent se terminer par l'extension `.js` ou `.ts` et le fichier source doit exister dans le dossier `src/pages/`.
+Les **Routes de Fichiers** sont des fichiers de script qui se termine par l'extension `.js` ou `.ts` et sont présents dans le dossier `src/pages/`.
 
 Les fichiers générés sont basés sur le nom du fichier source, ex: le résultat de la compilation de `src/pages/data.json.ts` correspondra à la route `/data.json` dans votre build final.
 
-En mode SSR (_server-side rendering_) l'extension importe peu et peut être omise, car le fichier n'est pas généré à la compilation.
+En mode SSR (_server-side rendering_) l'extension importe peu et peut être omise. Car aucun fichier n'est généré à la compilation. À la place, Astro génères un seul fichier sur le serveur.
```

Comment:
```suggestion
En mode SSR (_server-side rendering_) l'extension importe peu et peut être omise, car aucun fichier n'est généré à la compilation. À la place, Astro génères un seul fichier sur le serveur.
```
I think this should still be one sentence? It seems to be analogous to the English where it is one sentence. Can you double check this?